tree_sitter_v/grammar.js

```diff
@@ -83,7 +83,7 @@ const list_separator = choice(semi, ',');
 module.exports = grammar({
 	name: 'v',
 
-	extras: ($) => [/\s/, $.line_comment, $.block_comment],
+	extras: ($) => [/[ \t]+|(\r?\n)+/, $.line_comment, $.block_comment],
```

Comment:
If possible, please explain why this change

Reply:
Why won't `\s+` work?  The added `+` seems to be the only real change, except that yours covers fewer characters...
```
The \s metacharacter matches a whitespace character.

Whitespace characters can be:

     * A space character
     * A tab character
     * A carriage return character
     * A new line character
     * A vertical tab character
     * A form feed character
```